src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { FilterType } from '../../types/FilterTyper';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filterBy: FilterType;
+  todosCounter: number;
+  handleSetFilter: (filter: FilterType) => void;
+  todos: Todo[];
+  handleDeleteCompletedTodos: () => void;
+
+};
+
+const Footer: React.FC<Props> = React.memo(
+  ({ filterBy, todosCounter, handleSetFilter, todos, handleDeleteCompletedTodos}) => {
+
+    return (
+      <footer className="todoapp__footer" data-cy="Footer">
+        <span className="todo-count" data-cy="TodosCounter">
+          {todosCounter} {todosCounter === 1 ? 'item left' : 'items left'}
+        </span>
+        {/* Active link should have the 'selected' class */}
+        <nav className="filter" data-cy="Filter">
+          <a
+            href="#/"
+            className={cn("filter__link", {
+              "selected": filterBy === FilterType.all,
+            })}
+            data-cy="FilterLinkAll"
+            onClick={() => handleSetFilter(FilterType.all)}
+          >
+            All
+          </a>
+
+          <a
+            href="#/active"
+            className={cn("filter__link", {
+              "selected": filterBy === FilterType.active,
+            })}
+            data-cy="FilterLinkActive"
+            onClick={() => handleSetFilter(FilterType.active)}
+          >
+            Active
+          </a>
+
+          <a
+            href="#/completed"
+            className={cn("filter__link", {
+              "selected": filterBy === FilterType.completed,
+            })}
+            data-cy="FilterLinkCompleted"
+            onClick={() => handleSetFilter(FilterType.completed)}
+          >
+            Completed
+          </a>
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method

Reply:
Object.values(FilterType) is used  to dynamically render the filter options.